regression/pages/studio/studio_home.py

```diff
@@ -48,3 +48,17 @@ def click_view_live_button(self):
             "document.querySelectorAll('[data-course-key = \"course-v1:"
             "ArbiRaees+AR-1000+fall\"] .view-button')[0].click();")
         self.browser.switch_to_window(self.browser.window_handles[-1])
+
+    def click_terms_of_service(self):
+        """
+        Clicks Terms of Service link
+        """
+        self.q(css='a[href="https://stage.edx.org/edx-terms-service"]').click()
```

Comment:
My two goals are:
* Reduce repetition (make it DRY)
* Allow the environment to be configurable

So, I suggest a `BASE_STUDIO_URL` or something like that.